<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>normalWindows</class>
 <widget class="QWidget" name="normalWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您好，用户！</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="backCommand">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>91</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>查询车次</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>个人信息</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>购票记录</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>在线客服</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>186</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditdeparture"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>终点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditdestination"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditdate"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonresearch">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>411</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <column>
      <property name="text">
       <string>车次</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型一</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>余票数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型二</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>余票数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型三</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>余票数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButtonCheck">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>221</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>新用户名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditRepassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>新密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditNewpassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNewname"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>再次输入新密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>-100</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>车次</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>价格</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4"/>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
